assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -30,14 +30,23 @@ import Data from 'googlesitekit-data';
 import { createFetchStore } from '../../data/create-fetch-store';
 import { STORE_NAME } from './constants';
 import featureTours from '../../../feature-tours';
+import { setItem, getItem } from '../../../googlesitekit/api/cache';
+
 const { createRegistrySelector, createRegistryControl } = Data;
 const { getRegistry } = Data.commonActions;
 
+// Feature tour cooldown period is 2 hours
+export const FEATURE_TOUR_COOLDOWN_PERIOD = 1000 * 60 * 60 * 2;
+export const FEATURE_TOUR_CACHE_KEY = 'feature_tour_last_dismissed_at';
```

Comment:
Let's rename this to be more specific to the cache value for the last dismissed at timestamp. The actual cache key is currently more descriptive than it's constant 😄